python/caffe/draw.py

```diff
@@ -91,11 +91,11 @@ def get_layer_label(layer, rankdir):
                       separator,
                       layer.type,
                       separator,
-                      layer.convolution_param.kernel_size[0] if len(layer.convolution_param.kernel_size._values) else 1,
+                      layer.convolution_param.kernel_size[0] if len(layer.convolution_param.kernel_size) else 1,
                       separator,
-                      layer.convolution_param.stride[0] if len(layer.convolution_param.stride._values) else 1,
+                      layer.convolution_param.stride[0] if len(layer.convolution_param.stride) else 1,
                       separator,
-                      layer.convolution_param.pad[0] if len(layer.convolution_param.pad._values) else 0)
+                      layer.convolution_param.pad[0] if len(layer.convolution_param.pad) else 0)
```

Comment:
I'm getting `AttributeError: 'google.protobuf.pyext._message.RepeatedScalarConta' object has no attribute '_values'`  when trying with the included example nets. 

This happens when using `protobuf>2.6.1` (NVIDIA#303). I have tested the PR with `protobuf==2.5.0` and it works fine, just in case.